package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "realtime-currency-convertor",
+	"version": "1.0.0",
+	"description": "Realtime Currency Convertor",
+	"dependencies": {},
+	"devDependencies": {
+		"eslint": "^6.2.0",
+		"eslint-config-standard": "^13.0.1",
+		"eslint-plugin-import": "^2.18.2",
+		"eslint-plugin-node": "^9.1.0",
+		"eslint-plugin-promise": "^4.2.1",
+		"eslint-plugin-standard": "^4.0.0"
+	},
+	"scripts": {
+		"lint": "eslint ./src --fix",
+		"test": "echo \"Error: no test specified\" && exit 1"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Nullifiers/Realtime-Currency-Convertor.git"
```

Comment:
Any idea why git+ is appended before the url?

Reply:
@therealshabi?

Reply:
This doesn't have any explanation, as far as I researched this may be used in ssh and all and I found the same URL form in many of the node package.json file, including the Smartprix assistant project. As this was added from `npm init` itself and not via any manual effort, I don't think so this has that much of importance and we can leave this as it is for now, this surely has some meaning which I am  unable to find exactly